packages/editor/src/lib/editor/Editor.ts

```diff
@@ -7961,77 +7889,62 @@ export class Editor extends EventEmitter<TLEventMap> {
 					}
 
 					this.updateShapes(
-						shapesToUpdate.map((shape) => {
-							const props = { ...shape.props, [key]: value }
-							if (key === 'color' && 'labelColor' in props) {
-								props.labelColor = 'black'
-							}
-
```

Comment:
i'm not 100% on what this logic was there for, but i've removed it as i don't see how `labelColor` could be anything other than `black` anyway. Any ideas?